integration-cli/docker_utils_test.go

```diff
@@ -406,20 +402,15 @@ func minimalBaseImage() string {
 }
 
 func getGoroutineNumber() (int, error) {
```

Comment:
failing tests are counting  leaked goroutines. I'm not sure what caused that. I changed getGoroutineNumber to use client.Info instead of SockRequest